src/components/content/link.tsx

```diff
@@ -53,7 +52,14 @@ export function isLegacyLink(_href: string) {
   )
 }
 
-export function Link({
+export function Link(props: LinkProps) {
+  return UnstyledLink({
+    ...props,
+    className: clsx(props.className, 'serlo-link'),
+  })
+}
+
+export function UnstyledLink({
```

Comment:
exporting multiple components from one file is forbidden by inyono law, sorry ;)